string robot_id
geometry_msgs/PoseStamped door_handle_pose
---
bool success
string message